packages/flame/lib/src/image_composition.dart

```diff
@@ -0,0 +1,160 @@
+import 'dart:async';
```

Comment:
Strange that it didn't detect that file as moved, but thought that it was re-created. Maybe you could try `git mv` on it?

Reply:
that's because we left another file with the same name behind; so it thought this was a copy+rewrite instead of a move+create

Reply:
```
$ git commit -am 'move image_composition'
[ps/tests1 12c37401] move image_composition
 4 files changed, 41 insertions(+), 202 deletions(-)
 rewrite packages/flame/lib/image_composition.dart (100%)
 copy packages/flame/lib/{ => src}/image_composition.dart (97%)
 rewrite packages/flame/test/image_composition_test.dart (82%)
```